### ISSUE TITLE
1.7.0 fails to compile on aarch64 with TensorRT support

### ISSUE THREAD

**Issue by thomasyoung** [tensorflow/tensorflow#18875]
JetsonTX2 with JetPack 3.2 which has Cuda9, Cudnn7 and TensorRT 3.0. Here is the content in .tf_configure.bazelrc file:
```
build --action_env PYTHON_BIN_PATH="/usr/bin/python"
build --action_env PYTHON_LIB_PATH="/usr/local/lib/python2.7/dist-packages"
build --force_python=py2
build --host_force_python=py2
build --python_path="/usr/bin/python"
build --define with_jemalloc=true
build --define with_gcp_support=true
build --define with_hdfs_support=true
build --define with_s3_support=true
build:kafka --define with_kafka_support=true
build:xla --define with_xla_support=true
build:gdr --define with_gdr_support=true
build:verbs --define with_verbs_support=true
build --action_env TF_NEED_OPENCL_SYCL="0"
build --action_env TF_NEED_CUDA="1"
build --action_env CUDA_TOOLKIT_PATH="/usr/local/cuda"
build --action_env TF_CUDA_VERSION="9.0"
build --action_env CUDNN_INSTALL_PATH="/usr/lib/aarch64-linux-gnu"
build --action_env TF_CUDNN_VERSION="7"
build --action_env TENSORRT_INSTALL_PATH="/usr/lib/aarch64-linux-gnu"
build --action_env TF_TENSORRT_VERSION="4.0.4"
build --action_env TF_CUDA_COMPUTE_CAPABILITIES="3.5,5.2"
build --action_env LD_LIBRARY_PATH="/usr/local/cuda-9.0/lib64:"
build --action_env TF_CUDA_CLANG="0"
build --action_env GCC_HOST_COMPILER_PATH="/usr/bin/gcc"
build --config=cuda
test --config=cuda
build --define grpc_no_ares=true
build:opt --copt=-march=native
build:opt --host_copt=-march=native
build:opt --define with_default_optimizations=true
build --copt=-DGEMMLOWP_ALLOW_SLOW_SCALAR_FALLBACK
build --host_copt=-DGEMMLOWP_ALLOW_SLOW_SCALAR_FALLBACK
```
When building pip package:
```
.........................
ERROR: Skipping 'tensorflow/tools/pip_package:build_pip_package': error loading package 'tensorflow/tools/pip_package': Encountered error while reading extension file 'build_defs.bzl': no such package '@local_config_tensorrt//': Traceback (most recent call last):
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 167
		_trt_lib_version(repository_ctx, trt_install_path)
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 77, in _trt_lib_version
		_find_trt_header_dir(repository_ctx, trt_install_path)
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 60, in _find_trt_header_dir
		str(repository_ctx.path(("%s/../incl...)
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 60, in str
		repository_ctx.path(("%s/../include" % trt_install_path)).realpath
/usr/lib/include (No such file or directory)
WARNING: Target pattern parsing failed.
ERROR: error loading package 'tensorflow/tools/pip_package': Encountered error while reading extension file 'build_defs.bzl': no such package '@local_config_tensorrt//': Traceback (most recent call last):
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 167
		_trt_lib_version(repository_ctx, trt_install_path)
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 77, in _trt_lib_version
		_find_trt_header_dir(repository_ctx, trt_install_path)
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 60, in _find_trt_header_dir
		str(repository_ctx.path(("%s/../incl...)
	File "/home/nvidia/sourcecode/tensorflow/third_party/tensorrt/tensorrt_configure.bzl", line 60, in str
		repository_ctx.path(("%s/../include" % trt_install_path)).realpath
/usr/lib/include (No such file or directory)
```
Without TensorRT support it compiles well.

**Comment by thomasyoung** [comment]
Believe this is fixed by #17409.

**Comment by oscarriddle** [comment]
I encountered this issue, looks like something wrong with the path to tensorrt.
Din't see any relationship with #17409.
